packages/itwin/tree-widget/src/types.ts

```diff
@@ -16,31 +17,33 @@ export interface IModelContentTreeProps
 }
 
 export type ModelTreeProps = Omit<
-ModelsTreeProps,
-| "iModel"
-| "activeView"
-| "width"
-| "height"
-| "filterInfo"
-| "onFilterApplied"
+  ModelsTreeProps,
+  | "iModel"
+  | "activeView"
+  | "width"
+  | "height"
+  | "filterInfo"
+  | "onFilterApplied"
 >;
 
 export type CategoriesTreeProps = Omit<
-CategoryTreeProps,
-| "iModel"
-| "activeView"
-| "width"
-| "height"
-| "filterInfo"
-| "onFilterApplied"
+  CategoryTreeProps,
+  | "iModel"
+  | "activeView"
+  | "width"
+  | "height"
+  | "filterInfo"
+  | "onFilterApplied"
 >;
 
 export type SpatialTreeProps = Omit<
-SpatialContainmentTreeProps,
-"iModel" | "width" | "height"
+  SpatialContainmentTreeProps,
+  "iModel" | "width" | "height"
 >;
 
 export interface TreeWidgetOptions {
+  defaultPanelLocation?: StagePanelLocation,
```

Comment:
im surprised it took this long for a request like this to come through...

should we instead just expose an obj for users to set their preferred AbstractWidgetProps ?

Reply:
These locations are not defined in AbstractWidgetProps, but allowing those to be set may also be a good addition.

Reply:
@aruniverse - since you own this repo I will let you resolve this if you are ok with response.